Scripts/StaminaSystem.gd

```diff
@@ -0,0 +1,43 @@
+extends Label
+
+onready var stamina_label: Label = get_node(".")
+onready var ElfStats = get_parent().get_parent().get_node("ElfStats")
+
+var current_stamina: float = 10
+var max_stamina: float = 10
+var auto_atack: bool = true
+var timer = null
+
+func _ready():
+	max_stamina = ElfStats.get_stat("stamina").value
+	current_stamina = max_stamina
+	set_process_input(true)
+
+func _input(event):
+	if event is InputEventMouseButton and event.pressed and event.button_index == 1:
```

Comment:
I think, it's better to set custom action in project settings and use Input.is_action_just_pressed("our_action_name")